source/MaterialXRenderGlsl/GlslProgram.cpp

```diff
@@ -1054,10 +1055,11 @@ int GlslProgram::mapTypeToOpenGLType(const TypeDesc* type)
 
 const GlslProgram::InputMap& GlslProgram::updateAttributesList()
 {
-    if (_attributeList.size() > 0)
-    {
-        return _attributeList;
-    }
+    //if (_attributeList.size() > 0)
+    //{
+    //    return _attributeList;
+    //}
```

Comment:
Is this commented code now safe to remove from GlslProgram::updateAttributesList?

Reply:
Yes. I forgot to remove the commented out code.
As the attribute list can be dynamically updated we cannot perform an early exit just by testing if the attribute list exists.